website/content/docs/commands/events.mdx

```diff
@@ -0,0 +1,105 @@
+yourself to shut it down.
+
+## Examples
+
+Subscribe to all events:
+
+```shell-session
+$ vault events subscribe '*'
+```
+
+Subscribe to all KV events:
+
+```shell-session
+$ vault events subscribe 'kv*'
+```
+
+Subscribe to all `kv-v2/data-write` events:
+
+```shell-session
+$ vault events subscribe kv-v2/data-write
+```
+
+Subscribe to all non-`data-write` KV events for the secret `secret/data/foo` in
+the current namespace as well as namespace `ns1`:
+
+```shell-session
+$ vault events subscribe -namespaces=ns1 -filter='data_path == secret/data/foo and operation != "data-write"' 'kv*'
+```
+
+## Usage
+
+The following flags are available in addition to the [standard set of
+flags](/vault/docs/commands) included on all commands.
+
+### Options
+
+- `-timeout`: `(duration: "")` - close the WebSocket automatically after the
+  specified duration.
+
+- `-filter` `(string: "")` - Filter expression used to select events to be sent
+  through the WebSocket.
+
+  See [Boundary's docs](/boundary/docs/concepts/filtering) on how these
+  expressions are evaluated and what options are available.
+
+  The following values are available in the filter expression:
+  - `event_type`: the event type, e.g., `kv-v2/data-write`.
+	- `operation`: the operation name that caused the event, e.g., `write`.
+	- `source_plugin_mount`: the mount of the plugin that produced the event,
+    e.g., `secret/`
+	- `data_path`: the API path that can be used to access the data of the secret related to the event, e.g., `secret/data/foo`
+	- `namespace`: the path of the namespace that created the event, e.g., `ns1/`
+
+  By default, the filter will be empty and will match all events that the
+  request has access to for the given event types specified in the request.
+
+  A filter could be as simple as `data_path == secret/data/foo` to specify that
+  only events that refer to the `data_path` of `secret/data/foo` should be sent
+  via the WebSocket.
+
+  A filter can also be more complex, for example:
+  ```
+  data_path == secret/data/foo and operation != write
+  ```
+
+  The filter is applied after policy checks.
+
+### Enterprise Options
+
+<EnterpriseAlert product="vault" />
+
+- `-namespaces` `(string)` - Additional child namespaces to subscribe to. Can
+  be repeated to add more namespace patterns. The namespace of the request is
+  automatically prepended, so specifying `ns2` when the request is in the `ns1`
+  namespace will result in subscribing to `ns1/ns2` (in addition to `ns1`).
+  Patterns can include `*` characters to indicate wildcards. The default is to
+  subscribe only to the request's namespace.
+
+  -> Note: A root token, or a token associated with entities and groups with
+     the appropriate policies, must be used for the event subscription
+     request to be able to subscribe to multiple namespaces. See
+     [this tutorial](/vault/tutorials/enterprise/namespaces) for an example
+     setup.
```

Comment:
```suggestion
<Note>

  To subscribe to events across multiple namespaces, you must provide a root
  token or a token associated with appropriate policies across all the targeted
  namespaces. Refer to the
  [Secure multi-tenancy with namespaces](/vault/tutorials/enterprise/namespaces)
  tutorial for configuring your Vault instance appropriately.

</Note>
```

Style correction: use new notation for asides, write in active voice, avoid generic anchor text